src/token/PendingAirdropId.js

```diff
@@ -33,10 +33,10 @@ export default class PendingAirdropId {
      * @param {object} props
      * @param {AccountId} [props.senderId]
      * @param {AccountId} [props.receiverId]
-     * @param {TokenId?} props.tokenId
-     * @param {NftId?} props.nftId
+     * @param {TokenId?} [props.tokenId]
+     * @param {NftId?} [props.nftId]
      */
-    constructor(props) {
+    constructor(props = {}) {
         this._senderId = null;
         this._receiverId = null;
         this._tokenId = null;
```

Comment:
Why not set nftId to null here as well?

Reply:
what do you mean? It's set to null in the same line 🤔

Reply:
Wasn't expanded sorry.